Topic_2/blog/src/main/java/com/mycompany/app/blog/Entry.java

```diff
@@ -0,0 +1,19 @@
+package com.mycompany.app.blog;
+
+public class Entry {
+	private static int idGenerator = 1;

```

Comment:
This is not an attibute from the class. You must use a singleton to generate id number.